packages/svelte-ux/src/lib/components/Card.svelte

```diff
@@ -48,23 +48,23 @@
   {/if}
 
   {#if title || subheading || $$slots.header}
-    <div class={cls('p-4', classes.headerContainer)}>
+    <div class={cls('p-4', settingsClasses.headerContainer, classes.headerContainer)}>
       <slot name="header">
-        <Header {title} {subheading} classes={classes.header} />
+        <Header {title} {subheading} classes={{ ...settingsClasses.header, ...classes.header }} />
```

Comment:
There is a `clsMerge()` util to properly handle class overrides (for example: `p-2` in `classes.header` should override `p-1` in `settingsClasses.header`).  See `SelectField` / `MultiSelectField` / `ThemeSwitch` for example usage